src/libratbag-data.c

```diff
@@ -281,6 +282,13 @@ ratbag_device_data_get_name(const struct ratbag_device_data *data)
 	return data->name;
 }
 
+enum ratbag_device_type
+ratbag_device_data_get_device_type(const struct ratbag_device_data *data)
```

Comment:
Do we need this? I think we only use this when we need to have an assert.

Reply:
all other properties have it so i added it here as well